### PR TITLE
[4.0] Refactor quicklinks CSS

### PR DESCRIPTION
**Pull request by ciar4n**
Pull Request for Issue # .

### Summary of Changes
Refactors the quicklinks CSS, removing redundant rules and general cleanup. Fixes some minor spacing inconsistencies. 

### Testing Instructions
Apply PR and run `npm run build:css` to compile the SCSS. Check quicklinks are displayed as before.

### Expected result AFTER applying this Pull Request
image

### CODE REVIEW

File: layouts/joomla/quickicons/icon.php [hunk 1]
```diff
@@ -16,10 +16,8 @@
 $onclick = empty($displayData['onclick']) ? '' : (' onclick="' . $displayData['onclick'] . '"');
 
 if (isset($displayData['ajaxurl'])) {
-	$size = 'small';
```
Comment (Quy) [inline layouts/joomla/quickicons/icon.php:19]: > PHP Notice:  Undefined variable: size in \layouts\joomla\quickicons\icon.php on line 63

Comment (ciar4n) [reply layouts/joomla/quickicons/icon.php:19]: Fixed... thx @Quy

**Comment by chmst** [comment]
The size had a meaning. If numbers are added, the icon must be smaller. 
Before Patch

grafik

After patch 

quickicon

**Comment by ciar4n** [comment]
Thanks @chmst .. how are the numbers added?

**Comment by ciar4n** [comment]
@chmst should be ok now

**Comment by ChristineWk** [comment]
> 
> 
> Thanks @chmst .. how are the numbers added?

https://issues.joomla.org/tracker/joomla-cms/32223

**Comment by chmst** [comment]
@ChristineWk @ciar4n Edit the module, For many  site icons it is possible to choose show / hide / show item with count. 

It's one of the hidden secrets of J4 ....

grafik

**Comment by chmst** [comment]
I have tested this item :white_check_mark: successfully on 966bf6d8d03247e8cf2b39f70060168983aebf3b<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32336">issues.joomla.org/tracker/joomla-cms/32336</a>.</sub>

**Comment by chmst** [comment]
Names and numbers are now black, what changes the look of the buttons. It does not affect function.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32336">issues.joomla.org/tracker/joomla-cms/32336</a>.</sub>

**Comment by richard67** [comment]
@ciar4n The scss linter complains: https://ci.joomla.org/joomla/joomla-cms/39938/1/20

```
administrator/templates/atum/scss/blocks/_quickicons.scss
  21:40  ✖  Unexpected whitespace at end of line                      no-eol-whitespace     
  23:10  ✖  Unexpected whitespace at end of line                      no-eol-whitespace     
  27:7   ✖  Expected "align-items" to come before "justify-content"   order/properties-order
  30:7   ✖  Expected "text-align" to come before "background-color"   order/properties-order
  31:7   ✖  Expected "color" to come before "text-align"              order/properties-order
  34:26  ✖  Unexpected whitespace at end of line                      no-eol-whitespace     
  52:9   ✖  Expected "color" to come before "text-decoration"         order/properties-order
  58:8   ✖  Unexpected whitespace at end of line                      no-eol-whitespace     
  63:8   ✖  Unexpected whitespace at end of line                      no-eol-whitespace     
  72:10  ✖  Unexpected whitespace at end of line                      no-eol-whitespace     
  78:8   ✖  Unexpected whitespace at end of line                      no-eol-whitespace     
  84:8   ✖  Unexpected whitespace at end of line                      no-eol-whitespace     
  92:8   ✖  Unexpected whitespace at end of line                      no-eol-whitespace     
 107:7   ✖  Expected "font-size" to come before "background-color"    order/properties-order
 112:6   ✖  Unexpected whitespace at end of line                      no-eol-whitespace     
 119:7   ✖  Expected indentation of 8 spaces                          indentation           
 120:7   ✖  Expected indentation of 8 spaces                          indentation           
 121:7   ✖  Expected indentation of 8 spaces                          indentation           
 122:9   ✖  Expected indentation of 10 spaces                         indentation           
 124:11  ✖  Expected nesting depth to be no more than 4               max-nesting-depth
```

**Comment by ciar4n** [comment]
Thank you

> Names and numbers are now black, what changes the look of the buttons. It does not affect function.

@chmst Names and number should be now displayed as before

> The scss linter complains: https://ci.joomla.org/joomla/joomla-cms/39938/1/20

@richard67 Should be good now

**Comment by chmst** [comment]
I have tested this item :white_check_mark: successfully on fbba7f8d104a29e20f646c89cf347d1c9e45024d<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32336">issues.joomla.org/tracker/joomla-cms/32336</a>.</sub>

**Comment by richard67** [comment]
As @chmst 's screenshot above shows, and I have the same result here, the quicklinks are much bigger at certain screen sizes as they are without the PR. Is that really intended? It's a matter of taste maybe, but in my opinion it wastes space.

**Comment by ciar4n** [comment]
@richard67 Are you referring to width?

**Comment by chmst** [comment]
After npm run build:css it sould be ok.  My latest test shows icons after PR the same as before PR, same size, same colour.

**Comment by richard67** [comment]
@ciar4n Yes, width.

@chmst Sure I did run npm.

What I saw is that at certain widths icons are same as before, but if you make width smaller or larger then, then you will see a difference. The changes from 3 to 2 to 1 quickicon in a row happens at different widths than before, and from my point of view too early when reducing from large to smaller width.

Not sure if I explain it well, am not a CSS guy.

**Comment by ciar4n** [comment]
@richard67 I undertstand, I have reduced the min-width of each item to 160px (previously 180px). This should be closer to what you have in mind?

**Comment by richard67** [comment]
@ciar4n Yes, closer to sizes before and still fixing spacing inconsistencies as described.

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 40fb068d5e100586bd3f9aba4cc45eefc57fd4f3<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32336">issues.joomla.org/tracker/joomla-cms/32336</a>.</sub>

**Comment by richard67** [comment]
@chmst Could you test again? And make sure you clear cache after npm, or force reload or switch off cache, because it definitely can't be 100% the same as before at all sizes. Thanks in advance.

**Comment by chmst** [comment]
I have tested this item :white_check_mark: successfully on 40fb068d5e100586bd3f9aba4cc45eefc57fd4f3<br /><br />It is ok for me on my rather small monitor every width looks good. <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32336">issues.joomla.org/tracker/joomla-cms/32336</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32336">issues.joomla.org/tracker/joomla-cms/32336</a>.</sub>

**Comment by infograf768** [comment]
Tks

**Pull request merged**